app/lib/govuk_notify_personalisation.rb

```diff
@@ -400,6 +392,10 @@ def vaccine_side_effects
 
   private
 
+  def administered_year_groups
```

Comment:
Minor nit, but it was blocking my understanding of the code. Why "administered"? Is it not simply "year_groups"?

Reply:
Perhaps something like `year_groups_in_programme_scope` or `eligible_year_groups` (one for the future maybe...)